ledger-tool/src/blockstore.rs

```diff
@@ -600,6 +600,14 @@ pub fn blockstore_subcommands<'a, 'b>(hidden: bool) -> Vec<App<'a, 'b>> {
 }
 
 pub fn blockstore_process_command(ledger_path: &Path, matches: &ArgMatches<'_>) {
+    do_blockstore_process_command(ledger_path, matches).unwrap_or_else(|err| {
+        eprintln!("{err:?}");
+        std::process::exit(1);
+    });
+}
+
+
+fn do_blockstore_process_command(ledger_path: &Path, matches: &ArgMatches<'_>) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Hmm, specific Error type maybe? Given how few non-`BlockstoreError` sites there are, it's worth considering whether we could just return `BlockstoreError` here.
If absolutely a no-go, it might be nice to introduce a `LedgerToolError` enum, that we could eventually expand to handle errors across the tool. What do you think?

Reply:
I like this idea; went ahead and added a `LedgerToolError`. Not much work given that `thiserror` does most of the heavy lifting